src/data-access/repositories/mongo/mongo-database.ts

```diff
@@ -3,9 +3,13 @@ import { Collection } from 'mongodb'
 import { DatabaseNotConnectedException } from '../../../model/exceptions/database-not-connected-exception'
 import { MongoId } from './mongo-entity-converter'
 
-// TODO: Move to ENV variables
 const MONGO_CONNECTION_STRING: string = process.env.MONGO_URL ?? 'mongodb://localhost:3001'
-const MONGO_DB_NAME: string = 'meteor'
+const MONGO_DB_NAME: string = getMongoDatabaseName()
+
+function getMongoDatabaseName(): string {
+  const mongoUrlPattern: RegExp = /^mongodb:\/\/\w+(:\d+)?\/(?<databaseName>[^/]+)/i
+  return mongoUrlPattern.exec(MONGO_CONNECTION_STRING)?.groups?.databaseName ?? 'meteor'
```

Comment:
So the database name is now part of the database connection string in env?

Reply:
Yes. I have updated the PR description to state this as well.